### DISCUSSION TITLE
Question about Multi-Track Audio Streaming with SRT

### DISCUSSION THREAD

**Discussion by abenmohamed** [bluenviron/mediamtx#2574] [Questions]
### Question

I am attempting to achieve multi-track audio streaming using the SRT (Secure Reliable Transport) protocol in the MediaMTX project. However, I've encountered an issue where I am only able to receive a single audio track despite my efforts. I would appreciate your guidance and expertise to help me understand if multi-track audio streaming is possible with SRT and, if so, how to achieve it.

**Comment by aler9** [comment]
Please reopen the question and attach the server version and details on how you published with SRT to the server, how you read from the server (with which protocol?) and what software did you use for both, as explicitly mentioned in the instructions shown to you when you created the question.
